### PR TITLE
Bug/resize hero section button size

### LINKED ISSUE #353: Bug: Resize hero section button for smaller sizes.
### Describe the bug.

The button size looks bigger wrt to the section layout for max-width: 460px

### Expected behavior.

Ideally, it should look how it looks at 363 px.

### Screenshots.

Screenshot 2022-08-21 at 11 50 50 PM

### Operating Environment

macOS 12.5

### PR DESCRIPTION
**Pull request by prerana1821**
# Pull Request Template

## Description

I fixed the button size which was looking bigger for max-width: 460px. 

Fixes #353 

## Type of change

- [x] Bug fix:  button size on hero section.

## Additional Context (Please include any Screenshots/gifs if relevant)

image

## Checklist:

- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [ ] I have made corresponding changes to the documentation.
- [ ] I have added corresponding tests.
- [x] Any dependent changes have been merged and published in downstream modules.
- [x] I have signed the commit message to agree to Developer Certificate of Origin (DCO) (to certify that you wrote or otherwise have the right to submit your contribution to the project.) by adding "--signoff" to my git commit command.

### CODE REVIEW

File: website/src/pages/index.js [hunk 1]
```diff
@@ -12,31 +12,37 @@ import Sponsors from '../components/Sponsors';
...
+              >
+                Watch the tutorials
+              </Link>
+            </button>
+          </div>
         </div>
+        <img
+          className={styles.heroImg}
+          src="static/img/logo-light.png"
+          alt={'Logo'}
```
Comment (akshay1502) [inline website/src/pages/index.js:40]: No need to put curly braces, can be removed.

File: website/src/pages/index.module.css [hunk 2]
```diff
@@ -25,52 +10,189 @@
...
+}
+
+.btn>a {
+  color: black !important;
+  font-size: 1.2rem;
+  font-weight: 600;
 }
 
-html[data-theme='dark'] li > a:hover {
+.hero__subtitle {}
```
Comment (akshay1502) [inline website/src/pages/index.module.css:83]: Could we delete it, as it's serving no purpose?

File: website/src/pages/index.module.css [hunk 3]
```diff
@@ -25,52 +10,189 @@
...
+
+html[data-theme='dark'] li>a:hover {
   text-decoration: none;
   color: #ecf0f1;
 }
 
-.buttons > a {
-  margin: 5px;
+.buttons>a {}
```
Comment (akshay1502) [inline website/src/pages/index.module.css:90]: Is there any purpose in keeping this @prerana1821?

File: website/src/pages/index.module.css [hunk 4]
```diff
@@ -25,52 +10,189 @@
...
+
+
+
+@media only screen and (min-width: 200px) and (max-width: 810px) {
+  .heroContainer {
+    /* padding: 20px 0; */
+    display: flex;
+    margin: 0px auto 0px auto;
```
Comment (akshay1502) [inline website/src/pages/index.module.css:98]: Can we short-hand property here?

File: website/src/pages/index.module.css [hunk 5]
```diff
@@ -25,52 +10,189 @@
...
 
-.buttons > a {
-  margin: 5px;
+.buttons>a {}
+
+
+
+@media only screen and (min-width: 200px) and (max-width: 810px) {
+  .heroContainer {
+    /* padding: 20px 0; */
```
Comment (akshay1502) [inline website/src/pages/index.module.css:96]: Let's remove comments. Not a good idea to have them in production.

File: website/src/pages/index.js (hunk 1 above)
Comment (prerana1821) [reply website/src/pages/index.js]: Hey, I mistakenly chose main as the base branch.
I have just made changes in only one file.

**Pull request closed without merging**